### ISSUE TITLE
"Factory" not exported in browser

### ISSUE THREAD

**Issue by joliss** [rosiejs/rosie#13]
In a browser environment, rosie.js only exports its global Factory object if it's executed at the top level scope. But when it's not, for instance because it's wrapped like so ...

```javascript
loader.register('rosie', function(
  // ... rosie.js contents ...
});
```

... then there is no way to get at the `Factory` object. That's because `Factory` is defined [using `var`](https://github.com/bkeepers/rosie/blob/8b5c4722e228142f06e52f090bdeaa7fdccdb4b6/src/rosie.js#L1) and [never exported to `window`](https://github.com/bkeepers/rosie/blob/8b5c4722e228142f06e52f090bdeaa7fdccdb4b6/src/rosie.js#L82).

Could we have some code that tacks it onto the global `window` object? (Not sure what the most idiomatic way is, or I would've sent a pull request.)

**Comment by eventualbuddha** [comment]
This is a good idea. I realize you've probably moved on to other things, @joliss, but this is fixed by #39. I expect it'll be released in v1.1.0.

**Comment by eventualbuddha** [comment]
This has been released in v1.1.0.
